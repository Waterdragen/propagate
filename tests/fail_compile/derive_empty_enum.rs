use propagate::Propagate;

#[derive(Propagate)]
enum MyEnum {}

fn main() {}
