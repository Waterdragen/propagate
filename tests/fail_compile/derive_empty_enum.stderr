error: `Propagate` cannot be derived for enums without fields
 --> tests/fail_compile/derive_empty_enum.rs:3:10
  |
3 | #[derive(Propagate)]
  |          ^^^^^^^^^
  |
  = note: this error originates in the derive macro `Propagate` (in Nightly builds, run with -Z macro-backtrace for more info)
